python/createXSworkspace.py

```diff
@@ -105,8 +105,10 @@ def createXSworkspace(obsName, channel, nBins, obsBin, observableBins, usecfacto
         # obsNameOrig = obsName
         obsBin_low = observableBins[obsBin]
         obsBin_high = observableBins[obsBin+1]
+	print "obsBin_low:  ", obsBin_low
         if obsBin_high == 'inf':
             obsBin_high_inf = 10000
+	    if obsName=='dEtaj1j2': obsBin_high_inf = 10
```

Comment:
why is this one needed? if it is 10 or 10k for the highest bin, it should not matter no?

Reply:
ummm in principle it should not. But with the existing number i.e. `10000` the obtained observed events in last bin became saturated enough to exceed to inclusive no. of events. So, I put a more realistic upper limit.